### PR TITLE
Moved to new build and ci system

### PR DESCRIPTION
**Pull request by tmbo**
- switch to poetry build system (replacing pip & requirements files)
- switch to github actions for ci builds (replacing travis)
- make sure versions in different locations are the same (git tag, version.py, pyproject.toml)
- drop py 3.5 for black

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,19 +11,20 @@
...
 
 Used and Supported by:
 
 [<img src="https://rasa.com/docs/_static/rasa_logo.svg" width="60">](https://github.com/RasaHQ/rasa)
 
 ## Quickstart
 
-To install `questionary`, simply use [pipenv](http://pipenv.org/) (or pip, of
-course):
+The `questionary` can be installed using `pip`:
```
Comment (tmbo) [inline README.md:24]:
```suggestion
`questionary` can be installed using `pip`:
```

**Comment by tmbo** [comment]
needs to be merged to run on master

**Comment by kiancross** [comment]
@tmbo It says this was merged into `master` but I can't see it?

**Comment by tmbo** [comment]
I've removed the commits, I was checking if there was an issue with the actions setup, but the issue was somewhere else. this or is outdated, the #67 is the right one.

**Pull request merged**